nix/modules/launcher.nix

```diff
@@ -19,41 +18,45 @@
 
         cargoExtraArgs = "--bin hc-launch";
 
-        buildInputs =
+        buildInputs = (with pkgs; [
+          openssl
+
+          # TODO: remove this once the features have been rearranged to use vendored sqlite
+          sqlite
```

Comment:
can be avoided by bumping holochain in the launcher to include these respectively:
* 0.1 https://github.com/holochain/holochain/commit/6de1fb217679bf0948769946652aebcea0df8876
* 0.2 https://github.com/holochain/holochain/commit/ca440c705eef2a2351a63d4da849ed4376c2222c